Management/src/ActuatorApi/Steeltoe.Samples.ActuatorApi.csproj

```diff
@@ -0,0 +1,68 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+
+    <!-- Do not override package/assembly/file version with the version from git metadata when using GitInfo. -->
+    <GitVersion>false</GitVersion>
+    <!-- Do not generate (unused) C# classes from git metadata when using GitInfo. -->
+    <GitThisAssembly>false</GitThisAssembly>
```

Comment:
The comments explain what the properties do, which can be looked up if needed. But it doesn't explain *why* we're doing this: it breaks `cf push` (more specifically, the `dotnet-core-buildpack`) without locally-built binaries.